css/main.css

```diff
@@ -130,6 +139,7 @@ header + section p {
 }
 .selling-points div {
     margin: 0 20px;
+    width: 33%;
```

Comment:
They're automatically that width thanks to flexbox. Maybe you want to modify the margin?